core/chains/evm/logpoller/orm_test.go

```diff
@@ -674,3 +674,133 @@ func BenchmarkLogs(b *testing.B) {
 		require.NoError(b, err)
 	}
 }
+
+func TestSelectLogsWithSigsExcluding(t *testing.T) {
```

Comment:
lets add some comments throughout this test indicating which cases we're covering here

Reply:
I don't think this test covers the block range feature or different addresses right?

Reply:
Added comment and more tests in [`e2f04b0` (#8870)](https://github.com/smartcontractkit/chainlink/pull/8870/commits/e2f04b04bf86327a1c1598d65d2cf4d33cfd0629)